tests/cypress/tests/standalone/readwrite_navigation_test.ts

```diff
@@ -0,0 +1,122 @@
+import DashTable from 'cypress/DashTable';
+import DOM from 'cypress/DOM';
+import Key from 'cypress/Key';
+
+import { ReadWriteModes } from 'demo/AppMode';
+
+Object.values(ReadWriteModes).forEach(mode => {
```

Comment:
Navigation tests that involve the input element -- testing caret, navigation after/while editing, etc.